scripts/generateSupportedBrowsersRegExp.ts

```diff
@@ -24,6 +24,7 @@ function emitFileContent(outFile: string, verify = false) {
        * This file is generated by \`npm run generate:supportedBrowsersRegExp\`
        * Any manual changes will be overridden.
        */
+      // eslint-disable-next-line
       const supportedBrowsersRegExp = ${regexp};
```

Comment:
With the updates to prettier/linting, without this, it will get stuck in a loop when trying to commit of needing to regenerate this, but if it's generated, then the linting complains. So, I disabled the linting just for this line to break it out of this loop.